src/lib/extensions.js

```diff
@@ -15,6 +15,7 @@ export const CODE_EXTENSIONS = [
     'css',
     'cxx',
     'diff',
+    'dwg',
```

Comment:
CODE_EXTENSIONS are also used by the TextLoader, I think it's possible that the wrong viewer might try to load a dwg depending on order of an array.

Reply:
Ah, so should this go in DOCUMENT_EXTENSIONS instead?

Reply:
sounds good. Can DWGs have multiple pages or are they single page file types?

Reply:
They can have multiple pages.